### PR TITLE
Bug with binding implicitly unwarapped optional (Xcode 8.3, Swift 3.1 support)

### PR DESCRIPTION
**Pull request by azubala**
What's up:
- changed object type to `Any?` in object binding to avoid issues with implicitly unwrapped optional
- added specs for binding implicitly unwrapped optional
- updated Cocoapods
- bumped version to `0.7.0`
- Updated `CHANGELOG`

---
Just for the record @Przytua has reported the bug about this Swift bug here: https://bugs.swift.org/browse/SR-4552

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
   let myOhterObject: MyClass? = injector[MyClass.self] as MyClass // This requires explicit casting
   ```
 
+### Changed:
```
Comment (Przytua) [inline CHANGELOG.md]: Please change to '### Fixed:'

File: Sources/Swifjection/Classes/Bindings/Bindings.swift [hunk 2]
```diff
@@ -43,7 +43,7 @@ open class Bindings {
      - Parameter object: An instance that should be bound to provided `type`.
      - Parameter type: The `type` to which the `object` should be bound.
      */
-    public func bind(object: Any, toType type: Any.Type) {
+    public func bind(object: Any?, toType type: Any.Type) {
```
Comment (Przytua) [inline Sources/Swifjection/Classes/Bindings/Bindings.swift]: Please add check if 'object' is not nil, otherwise there's no point in adding binding.

**Review by Przytua — CHANGES_REQUESTED** [review]
Good one 👍 Just 2 remarks

**Comment by azubala** [comment]
@Przytua ready for the review.

**Review by Przytua — APPROVED** [review]
👍

**Pull request merged**